spec/acceptance/suites/default/10_proc_hidepid_spec.rb

```diff
@@ -5,43 +5,49 @@
 test_name 'polkit with /proc hidepid=2'
 
 describe 'polkit with /proc hidepid=2' do
-  hosts.each do |_host|
+  hosts.each do |host|
     let(:manifest) do
       'include polkit'
     end
+    context "Set hidepid but not gid on /proc on #{host}" do
+      it 'remounts /proc with hidepid=2' do
+        # if the gid is set then it does not get cleared unless rebooted
+        # The fstab should not have gid or hidepid set at this point.
+        host.reboot
+        on(host, 'mount -o remount,hidepid=2 /proc')
+      end
 
-    hosts.each do |host|
-      context "on #{host}" do
-        it 'remounts /proc with hidepid=2' do
-          on(host, 'mount -o remount,hidepid=2 /proc')
-        end
-
-        it 'shows a notification message but does not restart the service' do
-          output = apply_manifest_on(host, manifest, :accept_all_exit_codes => true).output
+      it 'shows a notification message but does not restart the service' do
+        output = apply_manifest_on(host, manifest, :accept_all_exit_codes => true).output
+        expect(output).to match(/hidepid warning/)
+        expect(output).not_to match(/Service/)
+      end
 
-          expect(output).to match(/hidepid warning/)
-          expect(output).not_to match(/Service/)
-        end
+      it 'does not show pkttyagent warnings when running service restarts' do
```

Comment:
It may be helpful to reference the ticket that describes what the problem was.  This will help code maintainers understand what the test was about and if it is still relevant.